website/docusaurus.config.js

```diff
@@ -356,7 +352,7 @@ module.exports = {
                     },
                     {
                         from: '/advanced/archived_toggles',
-                        to: '/reference/archived-toggles',
+                        to: '/reference/feature-toggles',
```

Comment:
We should probably update all the relevant redirects to target the new page.

It'll be (I think):

```json
from: ['/advanced/archived_toggles', '/reference/archived-toggles']
to: '/reference/feature-toggles'
```

Also, don't we do this server-side now? 🤔 

So this goes for this one and all the other deleted files (I noticed there's only two redirects here, but you've deleted at least 6 files)

Reply:
Ah, but I see you've also updated the vercel file, which should take care of it. In that case, are the changes here still necessary?

Reply:
Yes, I added the new redirects in Vercel, these are some legacy ones. I am not sure if these `'/advanced'` pages are still getting any traffic. Eventually, all redirects should be in Vercel, but I wanted to tackle this separately, as we might not need to move all of them.

> We should probably update all the relevant redirects to target the new page.
I am pointing these legacy pages to the new page too, unless I am missing something 🤔